Wrong text for assertTrue

I still have the same problem with the fault description of assertEquals and assertTrue, that was supposed to be solved.

My pom.xml:


```xml
        <!-- https://mvnrepository.com/artifact/org.testng/testng -->
        <dependency>
            <groupId>org.testng</groupId>
            <artifactId>testng</artifactId>
            <version>7.0.0-beta3</version>
            <scope>test</scope>
        </dependency>
```


My test file:

```java
import org.testng.annotations.Test;

import static org.testng.Assert.assertTrue;

public class testTestNg {
    @Test
    public void showFail() {
        assertTrue(false, "MyMessage");
    }

    @Test(expectedExceptions = AssertionError.class, expectedExceptionsMessageRegExp = "MyMessage did not expect to find \\[true\\] but found \\[false\\]")
    public void version7Behavior() {
        assertTrue(false, "MyMessage");
    }

    @Test(expectedExceptions = AssertionError.class, expectedExceptionsMessageRegExp = "MyMessage expected \\[true\\] but found \\[false\\]")
    public void version6Behavior() {
        assertTrue(false, "MyMessage");
    }
}
```


I the test "showFail", I get:
`java.lang.AssertionError: MyMessage did not expect to find [true] but found [false]`

which is bad language. Isn't that exactly what this issue is about?

_Originally posted by @HelgeStenstrom in https://github.com/cbeust/testng/pull/1936#issuecomment-492800979_

--- a/CHANGES.txt
+++ b/CHANGES.txt
@@ -1,4 +1,5 @@
 Current
+Fixed: GITHUB-2080: Wrong text for assertTrue (Jiong Fu)
 Fixed: GITHUB-2121: SoftAssert: allow custom message (dr29bart)
 Fixed: GITHUB-2110: NPE is thrown when running with Thread affinity (Krishnan Mahadevan)
 Fixed: GITHUB-2069: JUnit TestSuite not handled correctly in Reports (Krishnan Mahadevan)

--- a/src/main/java/org/testng/internal/EclipseInterface.java
+++ b/src/main/java/org/testng/internal/EclipseInterface.java
@@ -15,5 +15,5 @@ public class EclipseInterface {
   public static final String ASSERT_LEFT2 = "expected not same " + OPENING_CHARACTER;
   public static final String ASSERT_MIDDLE = CLOSING_CHARACTER + " but found " + OPENING_CHARACTER;
   public static final String ASSERT_RIGHT = Character.toString(CLOSING_CHARACTER);
-  public static final String ASSERT_LEFT_INEQUALITY = "did not expect to find " + OPENING_CHARACTER;
+  public static final String ASSERT_LEFT_INEQUALITY = "did expect " + OPENING_CHARACTER;
 }

--- a/src/test/java/org/testng/AssertTest.java
+++ b/src/test/java/org/testng/AssertTest.java
@@ -259,8 +259,8 @@ public class AssertTest {
     Assert.assertEquals("x", "y");
   }
 
-  @Test(description = "GITHUB-1935", expectedExceptions = AssertionError.class,
-      expectedExceptionsMessageRegExp = "did not expect to find \\[x\\] but found \\[x\\]")
+  @Test(description = "GITHUB-2080", expectedExceptions = AssertionError.class,
+      expectedExceptionsMessageRegExp = "did expect \\[x\\] but found \\[x\\]")
   public void testEqualityMessage() {
     Assert.assertNotEquals("x", "x");
   }

